app/views/users/_user.html.erb

```diff
@@ -50,19 +50,21 @@
       </div>
 
       <div class="col">
-        <%=
-          link_to "#",
-            data: { 
-              toggle: "modal", 
-              target: "##{dom_id(user)}_pending"
-            } do %>
-
-          <span class="font-weight-bold">
-            <%= user.pending.size %>
-          </span>
-          
-          pending
+        <% if current_user.username == @user.username%>
```

Comment:
I think here it should be user instead of @user to be consistent with the partial template